xep-0369.xml

```diff
@@ -580,28 +588,31 @@
+       A Nick MAY be associated with a participant, which provides a user-oriented  description of the participant.  The nick associated with the user is stored in a &lt;nick/&gt; child element of the &lt;participant/&gt; element.   The nick for each channel participant MUST be different to the nick of other participants.
+      </p>
+      <p>
+        Where a nick is provided for a user, it is generally recommended to use this nick or the JID to display the user.   This enables consistent representation of participants for all participants in the channel.
       </p>
       <p>
         The real JID of the user MAY be held in the participants node.   When the real JID is not present, the procedures defined in MIX-ANON must be followed.
         The user's JID is stored in a &lt;jid/&gt; child element of the &lt;participant/&gt; element.
       </p>
       <p>
-        When a user joins a channel, the user's bare proxy JID is added to the participants node by the MIX service.   When a user leaves a channel, the user's bare proxy JID is removed from the participants node.  The participants node MUST NOT be directly modified using pubsub.
+        When a user joins a channel, an item representing the is added to the participants node by the MIX service.   When a user leaves a channel, the user's item is removed from the participants node.  The participants node MUST NOT be directly modified using pubsub.
       </p>
       <p>
-        Clients will need to read the Participants node to provide useful display of presence information.  For this reason, it is anticipated that Participants will be able to read the Participants node.   However, for the message distribution specified in MIX-CORE, it is not necessary for clients to be able to read the Participants node.
+       It may be useful for clients to read the participants list.   However it is not necessary for message and presence display, as both messages and presence contain sufficient information to enable display.
       </p>
       <p>
         Users MAY subscribe to and read information this node, when permitted by the channel.  Standard PubSub access will allow a full list of participants and associated nicks to be determined.  By subscribing to the node, a user will be informed of changes to the Participants Node.
       </p>
-      <p>The participants node is OPTIONAL.  If the Participants Node is not used, information on channel participants is not shared.  If there is no participants node, the access control value 'participants' MUST NOT be used.   The Participants node is a permanent node with one item per participant.</p>
+      <p>The participants node is MANDATORY.   The Participants node is a permanent node with one item per participant.</p>
```

Comment:
This is a change in normative wording. I will bump the version number to 0.12 since this is non-editorial.